components/ProjectSummery.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { FontAwesome } from '@expo/vector-icons';
+
+const ProjectSummery = () => {
+
+    const ProjectDetails = styled.View`
+    margin: 21px 41px 0px 21px;
+    border: ${props => `2px solid ${props.theme.colors.staBlack}`}
+    padding: 17px 27px 11px 27px
+    `
+
+
+    const ProjectHeading = styled.View`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    `
+
+    const ProjectTitle = styled.Text`
+    font-weight: 700;
+    font-size: 18px;
+    `
+    const ProjectSubTitle = styled.Text`
+    font-weight: 400;
+    font-size: 16px;
+    `
+
+    const ProjectRole = styled.Text`
+    font-weight: 600;
+    font-size: 16px;
+    margin-top: 9px;
+    `
+
+    const ProjectDescription = styled.Text`
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+    `
+
```

Comment:
styled components should be outside of the component definition as they need to be static